Tests/SFSafeSymbolsTests/ImageExtensionTests.swift

```diff
@@ -0,0 +1,19 @@
+@testable import SFSafeSymbols
+import XCTest
+
+#if canImport(SwiftUI)
```

Comment:
If we check for this, we should also wrap the entire test into the `canImport` statement and let the test fail if `SwiftUI` cannot be imported. This way we get a test failure instead of a compiler error.